src/comm/LinkManager.cc

```diff
@@ -479,6 +479,28 @@ void LinkManager::_updateAutoConnectLinks(void)
         createConnectedLink(config);
         emit linkConfigurationsChanged();
     }
+#ifndef __mobile__
+    // check to see if nmea gps is configured for UDP input, if so, set it up to connect
+    if (_autoConnectSettings->autoConnectNmeaPort()->cookedValueString() == "UDP Port")
+    {
+        if (_nmeaSocket.localPort() != _autoConnectSettings->nmeaUdpPort()->cookedValue().toUInt()
```

Comment:
Internal C++ code would normally use rawValue and not cookedValue. cookedValues are normally for display to the user. Is there are reason why cookedValue is used here?

Reply:
my mistake, just fixed it.